prepare

```diff
@@ -64,5 +64,5 @@ REPOS_FILE="${REPOS_FILE:-sources.repos}"
 
 echo "Checkout workspace"
 
-mkdir src
-vcs import --recursive --input  "$REPOS_FILE" src
+mkdir -p src
+test -n "$SKIP_VCS_IMPORT" || vcs import --recursive --input  "$REPOS_FILE" src
```

Comment:
Could you use `if ...` instead of `||` for readability?

Reply:
Thanks! Updated!